app/src/main/java/co/smartreceipts/android/sync/widget/backups/ImportRemoteBackupWorkerProgressDialogFragment.java

```
@@ -111,7 +111,11 @@ public void onResume() {
                     }
                 }, throwable -> {
                     analytics.record(new ErrorEvent(ImportRemoteBackupWorkerProgressDialogFragment.this, throwable));
-                    Toast.makeText(getActivity(), getString(R.string.IMPORT_ERROR), Toast.LENGTH_LONG).show();
+                    if (throwable.getMessage().contains("More than five missing files")) {
```

For this, let's change it to actually check the exception type (and we can return a different exception for these two cases). This version won't be as sustainable (if we ever change the messaging)